internal/server/repository_worker.go

```diff
@@ -597,6 +597,7 @@ func (r *Repository) CreateWorker(ctx context.Context, worker *Worker, opt ...Op
 	}
 
 	worker.OperationalState = UnknownOperationalState.String()
+	worker.LocalStorageState = UnknownLocalStorageState.String()
```

Comment:
We should add a unit test in the internal/server/repository_worker_test.go file for the `TestRepository_CreateWorker` method. Something similar to the `success-with-state` test case.